task2.py

```diff
@@ -0,0 +1,8 @@
+min_in = int(input())
+H_in = int(input())
+M_in = int(input())
+min_total = min_in+H_in*60+M_in
```

Comment:
- better to separate each operation with space `H_in * 60 `
- define constant `min_per_hour = 60`